tidb-lightning/tidb-lightning-backends.md

```diff
@@ -0,0 +1,465 @@
+---
+title: TiDB Lightning Import Modes
+summary: Learn the different import modes of TiDB Lightning.
+aliases: ['/docs/dev/tidb-lightning/tidb-lightning-tidb-backend/','/docs/dev/reference/tools/tidb-lightning/tidb-backend/','/tidb/dev/tidb-lightning-tidb-backend']
+---
+
+# TiDB Lightning Import Modes
+
+TiDB Lightning supports three backends: Importer-backend (default), Local-backend, and TiDB-backend. It determines how `tidb-lightning` delivers data into the target cluster.
```

Comment:
```suggestion
TiDB Lightning supports three backends (import modes): Importer-backend (default), Local-backend, and TiDB-backend. The backend determines how `tidb-lightning` delivers data into the target cluster.
```
Please help confirm in this context: backends = import modes? @TomShawn

Reply:
@glorv PTAL

Reply:
Yes, I think `import mode` or something similar is a better way to express these three different kind of running type. `backend` is not easy for user to understand.

BTW, because the former version of this doc use `backend` to describe the difference, so when add `local` mode, I just followed it.

@kennytm  PTAL

Reply:
@glorv @TomShawn @yikeke  "import mode" is already used together with "normal mode". so it is bad to reuse the same terminology for something entirely different (backend).

Reply:
Makes sense. @kennytm I found the two terms in our glossary: [back end](https://docs.pingcap.com/tidb/dev/tidb-lightning-glossary#back-end) and [import mode](https://docs.pingcap.com/tidb/dev/tidb-lightning-glossary#import-mode). I think it'll be clearer if we give a reference link to this terminology at the beginning. @TomShawn 

Also, I think we should change "导入模式" to "后端" as it was introduced in [this PR](https://github.com/pingcap/docs-cn/pull/3979/files#diff-321248b8515b109cff5da454ea863ee7R2). @TomShawn

Reply:
Done in https://github.com/pingcap/docs/pull/3660/commits/16cb7f55cb2cc43dff87b82970498f4d49c02685
@yikeke @kennytm PTAL